### DISCUSSION TITLE
Change existing custom field

### DISCUSSION THREAD

**Discussion by EAleg** [netbox-community/netbox#6405] [Q&A]
### NetBox version

v2.10.5

### Feature type

Change to existing functionality

### Proposed functionality

Good afternoon, I want to offer you a small change in the existing function, namely: when you create a new field in the admin panel, you specify where it should be, but there is one place where it would come in handy. That is, when you go to the Devices, then Cisco 3600 (example), and add components -> inventory item. In our organization, it is very in demand, and in addition to the serial number, I would like to see the inventory number. In case for Cisco 3600 this inventory number for power supply.

The problem is that there is a field for Cisco, but for the inventory item it is not, the number can be added to the description, but the Search will not find anyway.

I would also like to see this field in the search menu on the right (Devices), or when you enter the number in the main search so that it finds it.

This is Cisco (Devices) (Here it goes separately)

![изображение](https://user-images.githubusercontent.com/82929335/115762996-b7e6f100-a3ac-11eb-89b6-08cc84059c1d.png)

This is Power Supply (Inventory item) for Cisco

![изображение](https://user-images.githubusercontent.com/82929335/115762961-ac93c580-a3ac-11eb-8cf3-c36abb29dcb4.png)

### Use case

There are inventory items where the serial number is missing, and the availability of the inventory number is required.

### Database changes

Unlikely

### External dependencies

Unlikely

**Comment by jeremystretch** [comment]
Sorry, I'm not sure I follow what you're proposing. If this is for support of custom fields for inventory items, please note that this has already been implemented in NetBox v2.11. If not, could you please clarify what you're proposing?

**Comment by EAleg** [comment]
> Sorry, I'm not sure I follow what you're proposing. If this is for support of custom fields for inventory items, please note that this has already been implemented in NetBox v2.11. If not, could you please clarify what you're proposing?

jeremystretch, thank you for answer! OMG, yes, its it is still there, I just have an old version. But, the Search does not find this inventory number, or did I miss somewhere? Important, inventory number is not a device, the subject (example Power Supply) is in the device.


![изображение](https://user-images.githubusercontent.com/82929335/115961482-3a8fbd80-a51f-11eb-868d-5263b0b0f47d.png)

Both Search not found (in Devices also)

![изображение](https://user-images.githubusercontent.com/82929335/115961621-df11ff80-a51f-11eb-8003-24e2010c8052.png)


Heres configuration of my custom field

![изображение](https://user-images.githubusercontent.com/82929335/115961531-788ce180-a51f-11eb-9aa9-afea4cb6db67.png)

![изображение](https://user-images.githubusercontent.com/82929335/115961539-82164980-a51f-11eb-8578-eb8d2cd701eb.png)

**Comment by jeremystretch** [comment]
Maybe someone else can help out here as I have no idea what you're asking.

**Reply by candlerb** [reply]
I think he's asking for global search on custom fields

**Reply by EAleg** [reply]
> I think he's asking for global search on custom fields

Yes

**Reply by jeremystretch** [reply]
This was proposed in #6003; please see that issue for discussion.

**Reply by EAleg** [reply]
> This was proposed in #6003; please see that issue for discussion.

Yep Jeremy, u r right. Thank you and sorry. Good luck!
